pcap/dlt.h

```diff
@@ -1496,6 +1496,12 @@
  */
 #define DLT_NETANALYZER_NG	291
 
+/*
+ * Serial NCP (Network Co-Processor) protocol for Zigbee stack ZBOSS
+ * by DSR, https://dsr-iot.com/downloads
```

Comment:
SInce this is behind a IPR claim and click-through: 

1. Could you be really really specific as to which part/document actually describes the protocol?
2. Is there a diagram or packet header that could go into linktypes.html on the web site?